packages/core/test/combinator/multicast-test.js

```diff
@@ -0,0 +1,37 @@
+import { describe, it } from 'mocha'
+import { assert, eq } from '@briancavalier/assert'
+import { spy } from 'sinon'
+
+import { now } from '../../src/source/now'
+import { map } from '../../src/combinator/transform'
+import { multicast } from '../../src/combinator/multicast'
+import { observe } from '../helper/observe'
+
+describe('multicast', () => {
+  it('should be identity for already-multicasted stream', () => {
+    const s = multicast(now(1))
+    eq(s, multicast(s))
+  })
+
+  it('should call mapper function once when there are > 1 observer', () => {
+    const mapperSpy = spy()
+    const observer1Spy = spy()
+    const observer2Spy = spy()
+    const observer3Spy = spy()
+
+    const s = now({value: 'sentinel'})
+    const mapped = map(mapperSpy, s)
+    const multicasted = multicast(mapped)
+
+    const o1 = observe(observer1Spy, multicasted)
+    const o2 = observe(observer2Spy, multicasted)
+    const o3 = observe(observer3Spy, multicasted)
```

Comment:
It'd be nice to use `collectEvents` or `runEffects` instead of `observe`.  Maybe something like:

```js
// import { runEffects } from '../../src/runEffects'
// import { ticks } from '../helper/testEnv'
const scheduler = ticks(1)

const o1 = runEffects(tap(observerSpy1, multicasted), scheduler)
const o2 = runEffects(tap(observerSpy2, multicasted), scheduler)
const o3 = runEffects(tap(observerSpy3, multicasted), scheduler)
```

I didn't try it, but I think that's close.